.github/workflows/ci.yml

```diff
@@ -375,27 +375,34 @@ jobs:
       working-directory: rizin
     - name: Compile with meson
       run: |
-        CFLAGS="-static" LDFLAGS="-static" meson --prefix=${HOME}/.local --buildtype release --default-library static build
+        mkdir -p ${HOME}/rizin-static
+        CFLAGS="-static" LDFLAGS="-static" meson --prefix=${HOME}/rizin-static --buildtype release --default-library static build
         ninja -C build && ninja -C build install
+        tar -C ${HOME}/rizin-static --strip-components=3 --xz -cf rizin-static.tar.xz ${HOME}/rizin-static
```

Comment:
why this? isn't the created tarball already in good shape?

Reply:
It isn't

Reply:
Doesn't help anyway